Tutorial/Window Help/wh_mainhelp.csv

```diff
@@ -1,12 +1,12 @@
+text05#0,"""<yellow>Handy Chat Log Commands<c><br><br>- Read through chat log: <yellow>Page Up<c>/<yellow>Page Down<c><br>- Switch log channel: <yellow>Ctrl+Left Arrow<c>/<yellow>Ctrl+Right Arrow<c><br>- Switch recipient: <yellow>Alt+Left Arrow<c>/<yellow>Alt+Right Arrow<c><br>- Chat entry log: <yellow>Ctrl+Up Arrow<c>/<yellow>Ctrl+Down Arrow<c><br>- Switch to/from direct chat mode: <yellow>Scroll Lock<c>"""
 text06#0,"""<yellow>Chat String Assignment Commands (1/2)<c><br><br>- <green><lt>me<gt><c><br>Assigns your character's name.<br><br>- <green><lt>target<gt><c> or <green><lt>t<gt><c><br>Assigns the name of your current target.<br><br>- <green><lt>weapon<gt><c> or <green><lt>wp<gt><c><br>Assigns the name of your currently equipped weapon."""
 text07#0,"""<yellow>Chat String Assignment Commands (2/2)<c><br><br>- <green><lt>Photon Art<gt><c> or <green><lt>pa<gt><c><br>Assigns the name of the Photon Art you just used.<br><br>- <green><lt>Technique<gt><c> or <green><lt>tc<gt><c><br>Assigns the name of the Technique you just used.<br><br>- <green><lt>pos<gt><c><br>Assigns the current map coordinates. (This command can only be used while on quests.)"""
 text08#0,"""<yellow>Special Chat Strings<c><br><br>- <green><lt>basecolor<gt><c> or <green><lt>bc<gt><c><br> Assigns the color of the nearest base in Mining Base Defense."""
-text09#0,"""<yellow>Handy Commands During Combat<c><br><br>- Switch Weapon Palettes (equip weapons)<br><yellow>R<c> / <yellow>F<c><br><br>- Select/execute left or right Subpalette<br>Select with <yellow>G<c> / <yellow>H<c> and execute with <yellow>B<c><br><br>- Switch to a different Subpalette<br><yellow>G<c> / <yellow>H<c> while switching the main palette (see above)"""
-text10#0,"""<yellow>Handy Window Commands (1/2)<c><br><br>- Go to next page/previous page of a list<br><yellow>Page Down<c> / <yellow>Page Up<c><br><br>- Select and manipulate multiple items<br>Add each item with <yellow>Ctrl+Click<c> or add a range with <yellow>Shift+Click<c><br><br>- Switch between tabs<br><yellow>Ctrl+Page Up<c> / <yellow>Ctrl+Page Down<c>"""
+text09#0,"""<yellow>Handy Commands During Combat<c><br><br>- Switch Weapon Palettes (equip weapons)<br><yellow>R<c>/<yellow>F<c><br><br>- Select/execute left or right Subpalette<br>Select with <yellow>G<c>/<yellow>H<c> and execute with <yellow>B<c><br><br>- Switch to a different Subpalette<br><yellow>G<c>/<yellow>H<c> while switching the main palette (see above)"""
+text10#0,"""<yellow>Handy Window Commands (1/2)<c><br><br>- Go to next page/previous page of a list<br><yellow>Page Down<c>/<yellow>Page Up<c><br><br>- Select and manipulate multiple items<br>Add each item with <yellow>Ctrl+Click<c> or add a range with <yellow>Shift+Click<c><br><br>- Switch between tabs<br><yellow>Ctrl+Page Up<c>/<yellow>Ctrl+Page Down<c>"""
```

Comment:
I think these should be left as they were, since they're mostly button combos. It helps readability to leave the spaces in.